android/engine/src/test/java/org/smartregister/fhircore/engine/data/local/DefaultRepositoryTest.kt

```diff
@@ -806,6 +812,50 @@ class DefaultRepositoryTest : RobolectricTest() {
     Assert.assertEquals(CarePlan.CarePlanStatus.COMPLETED, carePlan.status)
   }
 
+  @Test
+  fun testFilterRelatedResourcesShouldReturnTrueIfExpressionIsProvided() {
```

Comment:
Update test name to be more descriptive e.g `testFilterRelatedResourcesShouldReturnTrueIfProvidedExpressionEvaluatesToTrue`